src/test/javascript/spec/directive/orion-filter.directive.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { OrionFilterDirective } from 'app/shared/orion/orion-filter.directive';
+import { isOrion } from 'app/shared/orion/orion';
+
+@Component({
+    selector: 'jhi-test-component',
+    template: '<div id="shown" jhiOrionFilter [showInOrionWindow]="true"></div><div id="hidden" jhiOrionFilter [showInOrionWindow]="false"></div>',
+})
+class TestComponent {}
+
+describe('OrionFilterDirective', () => {
+    let fixture: ComponentFixture<TestComponent>;
+    let debugElement: DebugElement;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [],
+            declarations: [TestComponent, OrionFilterDirective],
+            providers: [],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(TestComponent);
+                debugElement = fixture.debugElement;
+            });
+    });
+
+    it('should show/hide elements if isOrion is true', fakeAsync(() => {
+        // @ts-ignore
+        isOrion = true;
```

Comment:
How does this work since this is not connected to anything? Github even complains below.

Reply:
This is just a mess, because isOrion is an exported boolean. Since its a value and not a function, the normal mocks don't work. I tried to use jest.mock, but then I could only change the value to a constant and not change it from test to test. So I finally settled with bluntly ignoring the const and overwriting it directly.

The value itself is used by the directive to determine whether the elements should be shown. I guess the GitHub check fails to realize this is an exported value that can be used in unrelated files